jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfigCredentialRetriever.java

```diff
@@ -81,6 +85,20 @@ public Path getDockerConfigFile() {
     if (!Files.exists(dockerConfigFile)) {
       return Optional.empty();
     }
+
+    if (legacyConfigFormat) {
```

Comment:
potential value in adding comment here (or somewhere) that `legacyConfigFormat` = `auths{}` block of dockerConfig.

Reply:
I was able to simplify this a bit.